css/main.scss

```diff
@@ -99,5 +99,6 @@ $flagsImagePath: "../images/";
 @import 'modals/security/security';
 @import 'premeeting-screens';
 @import 'e2ee';
+@import 'responsive';
```

Comment:
I added this new responsive module to add inside things that we want to make adaptable to the size of the screen.

I'm making some assumptions, about what functionality is not available when the window is too narrow:

For example: The filmstrip is removed under 500px width, the "overflow" menu is removed too (so... in certain sizes, what you see, is what you get :). And I'm removing the video resolution label under 500px too.

I have some concerns about visualizing this in mobile, probably would improve a bit the visual experience, but at the same time is going to remove certain features. Maybe we can try to detect mobile and do not apply this on mobile devices.

Reply:
Thank you for this, this very much aligns with plans to make web UI on mobile better.  Since this would be removing certain features, I will give this a try and discuss with the product team for what is acceptable before merging this PR.

Reply:
Great, yes, my approach was to open the PR with the changes to discuss around this and find the right degree of removal, or trying to find a way to distinguish between mobile and desktop (it can be just a class at the body that allows us to use specific css in certain cases).